templates/snapshot-deployment-sources-stage.yml

```diff
@@ -0,0 +1,32 @@
+parameters:
```

Comment:
Is the file name for this template correct? It doesn't seem to correspond with the job/steps